src/Blocks/BlockEnchantingTable.h

```diff
@@ -43,7 +43,8 @@ class cBlockEnchantingTableHandler :
 				return false;
 			}
 			auto EnchantingTable = static_cast<cEnchantingTableEntity &>(a_Entity);
-			if (!EnchantingTable.GetCustomName().empty())
+		  	const auto & CustomName = EnchantingTable.GetCustomName();
+			if (!CustomName.empty())
 			{
 				WindowName = EnchantingTable.GetCustomName();
```

Comment:
Not quite, you are not using the CustomName temporary variable here.

Reply:
you could even auto CustomName = GetCustomName and then std::move(CustomName) xD